tipsNtricks.md

```diff
@@ -110,6 +110,7 @@
 7.  Pen.io (Write Blog without registration)
 
 8.  Rainy mood -  [*soundrown.com*](https://soundrown.com/) like sites
+                  enjoy youtube videos in rainymood follow the link: http://rainymood.com/howto.html
```

Comment:
@techei Could you please change the link to markdown format like `[youtube videos](http://rainymood.com/howto.html)`